inc/tickettask.class.php

```diff
@@ -83,7 +83,7 @@ function canViewItem() {
          return false;
       }
 
-      if (Session::haveRightsOr(self::$rightname, [parent::SEEPRIVATE, parent::SEEPUBLIC])) {
+      if (Session::haveRight(self::$rightname, parent::SEEPRIVATE)) {
```

Comment:
I've also added this change as the current check doesn't seem correct (unless I am missing something ?).

Reply:
Dunno if the original code is working or not; but it sounds correct. I'm not really ok with this change

Reply:
The original code is checking for either public or private right to allow acces to ALL tasks.

The suggested changes make it check only for private rights (this is the way it's done for followups in ITILFollowup::CanViewItem).

Reply:
I'm not sure followups and tasks can be handled in the same logic. Will need @orthagh point of view.

Reply:
Regarding the code of ItilFollowup, the changes seems ok.